app/actions/authActions.js

```diff
@@ -0,0 +1,45 @@
+import { CALL_API } from 'redux-api-middleware';
+
+import types from 'constants/ActionTypes';
+
+function fetchAuthState() {
+  const params = { t: +new Date() };
+  return {
+    [CALL_API]: {
+      types: [
+        types.API.AUTH_GET_REQUEST,
+        types.API.AUTH_GET_SUCCESS,
+        types.API.AUTH_GET_ERROR,
+      ],
+      endpoint: `/auth?t=${params.t}`,
```

Comment:
- [ ] Does the t param just exist to prevent caching? If so, wouldn't it be better to handle that in Express by adding headers that prevent caching?